data/ka'het outfits.txt

```diff
@@ -179,7 +179,7 @@ outfit "Ka'het Primary Cooling"
 	"outfit space" -14
 	"active cooling" 65
 	"cooling energy" 5.9
-	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some Het by itself."
+	description "This is the main cooler used by the Ka'het. Although it requires a lot of energy, it is very effective, capable of upholding some Het by itself."
```

Comment:
While you're changing this, I'm not sure that "upholding" is the right word to use here. Maybe "...it is capable of cooling entire Het shits by itself"